R/vendor.R

```diff
@@ -9,13 +9,14 @@
 #' cpp11 currently installed on your machine.
 #'
 #' If you choose to vendor the headers you should _remove_ `LinkingTo:
-#' cpp11` from your DESCRIPTION.
+#' cpp11` from your DESCRIPTION. This is done automatically by this function.
```

Comment:
Is this true?

Reply:
yes! I added the changes for that, because I am a dummy